review-tools/gitaddrev

```diff
@@ -103,18 +123,25 @@ foreach (@ARGV) {
     } elsif (/^--rmreviewers$/) {
         $rmrev = 1;
     } elsif (/^--myemail=(.+\@.+)$/) {
-	try_add_reviewer($1);
+        push @collect_possible_reviewers, $1;
     } elsif (/^--verbose$/) {
 	$verbose = 1;
     } elsif (/^--web$/) {
         $WHAT = 'web';
+        $min_authors = 1;
+        # openssl/web is governed by OMC
+        $min_otc = 0;
+        $min_omc = 1;
     } elsif (/--tools$/) {
-        $WHAT = 'tools'
+        $WHAT = 'tools';
+        $min_authors = 1;
```

Comment:
So in the tools repo we allow committers to be a reviewer? That probably makes sense, but I'm not sure if that's been made explicit anywhere?

Reply:
I could set `$min_otc = 2` for this one...

Reply:
Just like I did for https://github.com/openssl/tools/pull/114#discussion_r865748628, I went looking for policies regarding this, and indeed, OTC are the only authorized reviewers, which makes `$min_otc = 2` the proper fix

Reply:
Fixed